### PR TITLE
GRW-1857 / Fix / Header responsive css

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes
Remove usage of useBreakpoint in header component and replace with css media querys. 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1857]: https://hedvig.atlassian.net/browse/GRW-1857?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/Header/HeaderStyles.tsx [hunk 1]
```diff
@@ -40,23 +40,33 @@ export const TriggerIcon = styled(ChevronIcon)({
...
 })
 
-export const NavigationTrigger = styled(NavigationMenuPrimitive.Trigger)({
-  display: 'flex',
-  alignItems: 'center',
-  gap: '0.5rem',
-  ...focusableStyles,
+export const NavigationTriggerDesktop = styled(NavigationMenuPrimitive.Trigger)({
+  display: 'none',
+  [mq.md]: {
```
Comment (alebedev) [inline apps/store/src/components/Header/HeaderStyles.tsx:45]: Nit: can you extract repeated styles into variable?

File: apps/store/src/components/Header/TopMenuDesktop/TopMenuDesktop.tsx [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 import React from 'react'
+import { mq } from 'ui'
 import { Navigation } from '../HeaderStyles'
 import { NavigationPrimaryList } from '../HeaderStyles'
 
 export type TopMenuDesktopProps = {
   children: React.ReactNode
 }
 
+const Wrapper = styled.div({
```
Comment (alebedev) [inline apps/store/src/components/Header/TopMenuDesktop/TopMenuDesktop.tsx:11]: Nit: can we use styled(Navigation) to avoid extra wrapper?

**Review by alebedev — APPROVED** [review]
Nice!  Few nit comments

**Review by guilhermespopolin — APPROVED** [review]
Great :)
Just curious: what's the motivation behind those changes? Is it related with minimizing the usage of JS for things that can be done with CSS or something else? 😄

**Comment by amandaflod** [comment]
@guilhermespopolin Yep, that is the only reason! :) 
@alebedev Thanks for the nits! Have updated some stuff now 🤠

**Pull request merged**